components/user-info/frontend/src/view/user-info.html

```diff
@@ -0,0 +1,41 @@
+<div class="col-md-12 col-sm-12">
+  <div class="application-header font-semi-bold">
+    <span translate class="app-header-app-name">user-info</span>
+    <actions-toolbar ready="userCtrl.ready" items="userCtrl.actions"></actions-toolbar>
+  </div>
+
+  <div class="application-details col-sm-12">
```

Comment:
Would it be worth it to stick these in a summary 'card panel' as per cloud foundry pages? Maybe with a vnext task of putting the 'groups' into a tab and contents in a 'label, description' table?